### PR TITLE
Desig Wallet support

### PR DESCRIPTION
**Pull request by thiennv-qng**
## Description

_Concise description of proposed changes_

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

### CODE REVIEW

File: packages/connectors/src/types.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ type InjectedProviderFlags = {
   isXDEFI?: true
   isZerion?: true
   isHaloWallet?: true
+  isDesig?: true
```
Comment (tmm) [inline packages/connectors/src/types.ts:55]: Alphabetize

File: packages/connectors/src/utils/getInjectedName.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export function getInjectedName(ethereum?: WindowProvider) {
   if (!ethereum) return 'Injected'
 
   const getName = (provider: WindowProvider) => {
+    if (provider.isDesig) return 'Desig Wallet'
```
Comment (tmm) [inline packages/connectors/src/utils/getInjectedName.ts:7]: Alphabetize and add test case.

File: .changeset/shiny-files-applaud.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/connectors": patch
+---
+
+Desig Wallet support
```
Comment (tmm) [inline .changeset/shiny-files-applaud.md:5]:
```suggestion
Added Desig Wallet name mapping.
```

**Review by tmm — CHANGES_REQUESTED** [review]

File: packages/connectors/src/types.ts (hunk 1 above)
Comment (thiennv-qng) [reply packages/connectors/src/types.ts]: plz help me take a look again. tks

File: packages/connectors/src/utils/getInjectedName.ts (hunk 2 above)
Comment (thiennv-qng) [reply packages/connectors/src/utils/getInjectedName.ts]: plz help me take a look again. tks

File: .changeset/shiny-files-applaud.md (hunk 3 above)
Comment (thiennv-qng) [reply .changeset/shiny-files-applaud.md]: plz help me take a look again. tks

**Review by jxom — APPROVED** [review]

**Pull request merged**